com.unity.render-pipelines.universal/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - Added Adaptive Performance Decals scaler access.
 
+### Changed
+- Reverted intermediate texture behavior.
+
```

Comment:
We can't change this on backport PR. This entry has to be removed.

Reply:
I've moved it to `Fixed`